ccm_web/client/src/utils/canvasSectionNameValidator.ts

```diff
@@ -19,10 +19,19 @@ class DuplicateSectionNameValidator implements IValidator {
   }
 }
 
```

Comment:
Probably I did not notice this in with Bulk enroll user UI PR. Why is this file in `utils` folder? Since these are validation around create section widget. If you feel my suggestion is reasonable can you move this to components?

Reply:
I think it makes more sense in utils.
It's utility/common functionality that could be used other places.  It doesn't really have any component-ness to it  IMO.

Reply:
ok